integration/docker/package_manager_test.go

```diff
@@ -46,8 +45,25 @@ var _ = Describe("package manager apt-get", func() {
 
 	Context("check apt-get update", func() {
 		It("should not fail", func() {
+			args = append(args, "--rm", "--name", id, DebianImage, "apt-get", "update")
```

Comment:
Can you update the `Describe()` comment (line 25) to make it more generic ("package manager update test" or something?), since this test is now testing running `apt-get` *and* `dnf`.

Reply:
Also, could you please add ```-y```